### PR TITLE
Fix cut off icon on Camera Permission screen for Document Auto Capture

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
The icon on Camera Permission screen for Document Auto Capture appears to be cut off on the top and bottom. 
The same screen for Selfie, Liveness, Document Live Capture displays the icon in full.
Screenshot 2020-07-15 at 14 56 00

# Solution
* Remove redundant top level `<div>` in `DocumentAutoCapture` component as this causes the child elements with flex box styling to collapse height wise.
* Remove flexbox display only for Document Auto Capture screen as layout does not require it and positioning of the document capture frame is off without the top level `<div>`.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by kopijunkie** [comment]
> I thought I had already fixed this somewhere :) It's because of the composition, isn't it...

Yeap. It displays fine on all the other steps with the Camera Permission, so probably this one got left behind. :)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1116-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/index.js [hunk 1]
```diff
@@ -53,9 +54,10 @@ const CameraPure = ({
   buttonType,
   onButtonClick,
   isButtonDisabled,
-  hasGrantedPermission
+  hasGrantedPermission,
+  displayBlock
 }: Props) => (
-  <div className={classNames(style.camera, className)}>
+  <div className={classNames(style.camera, { [style.displayBlock]: displayBlock }, className)}>
```
Comment (josokinas) [inline src/components/Camera/index.js:60]: Can't it be by default on `.camera` class so prop would not be needed?

Comment (kopijunkie) [reply src/components/Camera/index.js:60]: Good point. I'll double check on a few screen sizes, devices to be sure but I think you are right.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1116-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1116-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**